Makefile

```diff
@@ -22,9 +22,13 @@ build: build-go
 build-go:
 	go build ./...
 
+.PHONY: rm-cli
+rm-cli:
+	rm -f ./bin/$(GOOS)-$(GOARCH)/viam-cli
+
 GOOS ?= $(shell go env GOOS)
```

Comment:
FYI: I double checked that it's fine for the reference to GOOS/GOARCH is allowed to come after the assignment.